docs/development/pull-requests.md

```diff
@@ -221,11 +221,12 @@ seem unfamiliar, refer to this
 
 #### Approval and Request Changes Workflow
 
-All pull requests require approval from a [Code Owner](https://github.com/orgs/electron/teams/code-owners) of the area you
-modified in order to land. Whenever a maintainer reviews a pull request they
-may request changes. These may be small, such as fixing a typo, or may involve
-substantive changes. Such requests are intended to be helpful, but at times
-may come across as abrupt or unhelpful, especially if they do not include
+All pull requests require approval from a 
```

Comment:
Linter says there's trailing whitespace here

Reply:
Linter was indeed correct. Thanks for the catch! 🙇